specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2023-09-01/virtualMachine.json

```diff
@@ -1891,6 +1891,331 @@
           "final-state-via": "location"
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Compute/virtualMachines/{vmName}/VMApplications": {
```

Comment:
>VMApplications [](http://example.com/codeflow?start=125&length=14)

virtualMachineApplications or just applications

Reply:
VMApplications is the product name